README.md

```diff
@@ -2071,10 +2070,12 @@ final class ValidTest extends TestCase
 | :information_source: **INFORMATION:** |
 |:--------------------------------------|
 
-- It's better to avoid a shared state between tests.
-- To reuse elements between a few tests:
-    * private factory methods - reusing in one class (like above)
-    * [Object mother](#object-mother) or [Builder](#builder) - reusing in a few classes
+The usage of private methods improves readability and clarifies separation. 
+In the same time, additional effort is needed to use them inside particular testing method.  
```

Comment:
Hmm, I don't get this sentence. Could you provide any explanation?

Reply:
I wanted to underline that when we use helpers, and configure the sut, we have to repeat each call in testing method, that could be sometime tedious if imo. But yeah, I can get rid of it if it's vague.

Reply:
Ok, could improve the grammar and maybe add something about optimizing the code for reading, because we read the code more often than we write it. And I think that's all, and I will merge this PR, thank you for this PR.